docs/diagrams/UpcomingMeetingDiagrams/UpcomingMeetingSequenceDiagram.puml

```diff
@@ -0,0 +1,78 @@
+@startuml
+!include ../style.puml
+
+box UI LOGIC_COLOR_T1
+participant ":MainWindow" as MainWindow LOGIC_COLOR
+participant ":MeetingsWindow" as MeetingsWindow LOGIC_COLOR
+participant ":ObservableList" as ObservableList LOGIC_COLOR
+participant ":MeetingsListPanel" as MeetingsListPanel LOGIC_COLOR
+participant ":MeetingCard" as MeetingCard LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Person" as Person MODEL_COLOR
+end box
+
+[-> MainWindow : f2
+activate MainWindow
+
+MainWindow -> MainWindow : handleMeetings()
+activate MainWindow
+
+MainWindow -> MeetingsWindow : getMeetings()
+activate MeetingsWindow
+
+create ObservableList
```

Comment:
Code to illustrate: ObservableList<Person> personObservableList = logic.getFilteredPersonList();
Should be:
MainWindow -> LogicManager:getFilteredPersonList()
LogicManager --> MainWindow:personObservableList

Participant to add: Logic Manager from Logic